contribs/gnogenesis/internal/txs/txs_add_packages.go

```diff
@@ -76,27 +89,28 @@ func execTxsAddPackages(
 		return errInvalidPackageDir
 	}
 
-	var (
-		creator = defaultCreator
-		err     error
-	)
+	signer, err := signerWithConfig(cfg, io, genesis.ChainID)
```

Comment:
Where do you explicitly check if the key exists in gnokey?

Reply:
I don't explicitly check it on the added code, but if the  key does not exists on gnokey It will return an error while getting the Signer.Info()
```go
info, err := signer.Info()
if err != nil {
	return fmt.Errorf("unable to get signer info, %w", err)
}
```

Reply:
Now changed to directly consult keybase on 09dce4322f68e8f7aa366c3c33da287d5761d8ae